scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -224,6 +224,8 @@ test-linux-stable:
       --features runtime-benchmarks,try-runtime,experimental
       --manifest-path ./bin/node/cli/Cargo.toml
       --partition count:${CI_NODE_INDEX}/${CI_NODE_TOTAL}
+    # run runtime-api tests with `enable-staging-api` feature
```

Comment:
Is this a good idea?

Reply:
I haven't marked the correct line. It doesn't run all tests with `enable-staging-api` but only `sp-api-test`:
```
time cargo nextest run -p sp-api-test --features enable-staging-api
```
I've got some runtime api unit tests with feature flags, but they need to be invoked individually.